pkg/sfu/streamallocator/streamallocator.go

```diff
@@ -559,9 +559,7 @@ func (s *StreamAllocator) maybePostEventAllocateTrack(downTrack *sfu.DownTrack)
 }
 
 func (s *StreamAllocator) postEvent(event Event) {
-	s.eventsQueue.Enqueue(func() {
-		s.handleEvent(&event)
-	})
+	s.eventsQueue.Enqueue(s.handleEvent, event)
 }
 
 func (s *StreamAllocator) ping() {
```

Comment:
this function is responsible for 10% of allocated objects in london

Reply:
oh boy